src/envo/envo-edit.owl

```diff
@@ -40125,6 +40127,37 @@ AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#inSubset> <htt
 AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_06105010> "plastic pellet"@en)
 SubClassOf(<http://purl.obolibrary.org/obo/ENVO_06105010> <http://purl.obolibrary.org/obo/ENVO_01000776>)
 
+# Class: <http://purl.obolibrary.org/obo/ENVO_06105014> (biodegradation)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <https://doi.org/10.1351%2FPAC-REC-10-12-04>) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_06105014> "A biological process during which substance or chemical is broken down due to the action of cells."@en)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/contributor> <http://purl.obolibrary.org/obo/ENVO_06105014> <https://orcid.org/0000-0002-2996-719X>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/contributor> <http://purl.obolibrary.org/obo/ENVO_06105014> <https://orcid.org/0000-0003-1072-8081>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.obolibrary.org/obo/ENVO_06105014> <http://orcid.org/0000-0002-8166-0486>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.obolibrary.org/obo/ENVO_06105014> "2021-05-06T12:15:46Z"^^xsd:dateTime)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynonym> <http://purl.obolibrary.org/obo/ENVO_06105014> "Biodegradation"@de)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynonym> <http://purl.obolibrary.org/obo/ENVO_06105014> "Biologischer Abbauprozess"@de)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynonym> <http://purl.obolibrary.org/obo/ENVO_06105014> "biodegradación"@es)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynonym> <http://purl.obolibrary.org/obo/ENVO_06105014> "biodegradazione"@it)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynonym> <http://purl.obolibrary.org/obo/ENVO_06105014> "生物分解"@zh)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#inSubset> <http://purl.obolibrary.org/obo/ENVO_06105014> "envoPlastics"@en)
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_06105014> "biodegradation"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_06105014> <http://purl.obolibrary.org/obo/GO_0008150>)
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_06105015> (biodegradation of plastic)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <https://doi.org/10.1016/j.matpr.2020.06.096>) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_06105015> "A biodegradation process during which the molecular weight of plastic is reduced."@en)
```

Comment:
>A biodegradation process during which the molecular weight of plastic is reduced

This departs from the logic you set up in the superclass. Consider:

>A biodegradation process during which plastic is broken down. 

and then add a comment about the reduction in molecular weight.